types/dialogflow/index.d.ts

```diff
@@ -840,10 +841,16 @@ export enum IntentView {
     INTENT_VIEW_FULL = "INTENT_VIEW_FULL"
 }
 
+export enum WebhookState {
```

Comment:
In the v2 version this enum is a number type
https://github.com/googleapis/nodejs-dialogflow/blob/d8c259ec1e24cf8412e8d6a79ac1e08ef11e57c5/src/v2/doc/google/cloud/dialogflow/v2/doc_intent.js#L825